aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -983,22 +968,17 @@ impl VMExecutor for AptosVM {
             ))
         });
 
-        let concurrency_level = Self::get_concurrency_level();
-        if concurrency_level > 1 {
-            let (result, err) = crate::parallel_executor::ParallelAptosVM::execute_block(
-                transactions,
-                state_view,
-                concurrency_level,
-            )?;
-            debug!("Parallel execution error {:?}", err);
-            Ok(result)
-        } else {
-            let output = Self::execute_block_and_keep_vm_status(transactions, state_view)?;
-            Ok(output
-                .into_iter()
-                .map(|(_vm_status, txn_output)| txn_output)
-                .collect())
-        }
+        // Record the histogram count for transactions per block.
+        BLOCK_TRANSACTION_COUNT.observe(transactions.len() as f64);
```

Comment:
uh, great catch, thanks a lot! I just verified and it was only reported in sequential mode and not in parallel (already problematic) so I guess I was trying to fix that, however, only in the success case, which I had missed the '?'. 

So let's keep the "on-success" behavior, but now it will work for both execution modes.